UCLALibrary/docusign
# SYS-1708: Configuration reconfiguration

Implements [SYS-1708](https://uclalibrary.atlassian.net/browse/SYS-1708).

This PR refactors demo code to be more robust.  Key changes:
* Secrets and (some) other configuration are now stored in a TOML file, instead of being directly imported.
  * This includes the private key, formerly stored in a separate file.
* Programs have a required `--config CONFIG_FILE` argument.
* `jwt_utils.py` is refactored to remove private key file reading, now getting all config info via `get_config()`.
* `README.md` is updated to document config file info.
* `form_data_demo.py` is renamed to `get_form_data.py`
* `jwt_utils.dump_form_data()` replaced by `get_form_data.get_form_data_from_envelope()`

Testing:
* Image does not need to be rebuilt.
* Delete `jwt_secrets.py` and `docusign_secret_private.key` previously used.
* Get `secrets_DEV.toml` from me.
* Run `get_form_data.py --config secrets_DEV.toml` as documented in README, and confirm it prints info from two envelopes.


[SYS-1708]: https://uclalibrary.atlassian.net/browse/SYS-1708?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
